apps/andi/lib/andi_web/live/dataset_live_view/table.ex

```diff
@@ -32,4 +36,11 @@ defmodule AndiWeb.DatasetLiveView.Table do
     send(self(), {:order, field})
     {:noreply, socket}
   end
+
+  defp ingest_status(dataset) do
+    case dataset["ingested_time"] do
```

Comment:
This should be pulled out to a function codifies this logic as meaning a dataset is ingested rather than being implicit within the template.